### PR TITLE
feat(root): Implement Checkbox and Radio components

### PR DESCRIPTION
**Pull request by mucahit**
### Description
- Checkbox, Radio, and RadioGroup components are done 🎉 
- I used `react-svg-loader` to import SVG's as React component 
- Added a custom TypeScript declaration file to load SVG files
- Added animation related SCSS variables in `_animations.scss`

**If you'd like to see how those components look like, then [check it out here](https://react-ui-toolkit.muco.now.sh/?path=/story/form--checkbox-states)**

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+  
+  config.module.rules.push({
+    test: /\.svg$/,
+    exclude: /node_modules/,
+    use: [
+      {
+        loader: require.resolve("babel-loader")
+      },
+      {
+        loader: require.resolve("react-svg-loader")
```
Comment (edizcelik) [inline .storybook/webpack.config.js:16]: How does it compare to `@svgr/webpack` ?

File: src/form/input/checkbox/Checkbox.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  inputProps: {
+    htmlFor: string;
+    value: string;
+    name: string;
+  };
+}
+
+export interface CheckboxInputProps {
+  item: CheckboxInputItem;
+  onSelect: (name: string, item: CheckboxInputItem) => void;
```
Comment (edizcelik) [inline src/form/input/checkbox/Checkbox.tsx:21]:
```suggestion
  onSelect: (item: CheckboxInputItem, event?: React.SyntheticEvent<HTMLInputElement>) => void;
```

Let's change the order and pass the whole event as the second argument

File: src/form/input/radio/Radio.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    name: string;
+  };
+  context?: any;
+  customClassName?: string;
+  isDisabled?: boolean;
+}
+
+export type RadioInputSelectHandler<Id = string> = (
+  name: RadioInputItem["inputProps"]["name"],
+  item: RadioInputItem<Id>
```
Comment (edizcelik) [inline src/form/input/radio/Radio.tsx:22]: Same here

File: src/form/input/radio/group/RadioGroup.tsx [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+          className={classNames("radio-group-item", item.customClassName)}>
+          <RadioInput
+            isDisabled={isDisabled || item.isDisabled}
+            item={item}
+            onSelect={onSelect}
+            isSelected={Boolean(selectedItem && selectedItem.id === item.id)}
+          />
+        </li>
+      ))}
+    </ul>
```
Comment (edizcelik) [inline src/form/input/radio/group/RadioGroup.tsx:41]: Can we use `List` and `ListItem` components ?

File: .storybook/webpack.config.js (hunk 1 above)
Comment (mucahit) [reply .storybook/webpack.config.js:16]: `@svgr/webpack` has rollup port but it's buggy. `react-svg-loader` is a nice alternative

**Pull request merged**